### DISCUSSION TITLE
TLS1.3 support if using mbedtls

### DISCUSSION THREAD

**Discussion by zydingjun** [curl/curl#10587] [Q&A] [answered]
The latest version 7.88.1 does not support TLS1.3 if using mbedtls library, is any plan to add tls1.3 support for vtls_mbedtls?

**Comment by bagder** [comment, marked as answer]
Feel most welcome to contribute whatever code is necessary to enable that.

We're an open source project, we're all developers.

**Comment by bagder** [comment]
See also #12876
